src/mbtest/imposters/stubs.py

```diff
@@ -47,3 +47,39 @@ def from_structure(structure: JsonStructure) -> "Stub":
             [Predicate.from_structure(predicate) for predicate in structure.get("predicates", ())],
             responses,
         )
+
+
+class AddStub(JsonSerializable):
```

Comment:
Any reason we can't we use the standard Stub class here?

Reply:
AddStub request looks like:
`{
  "index": 0,
  "stub":<your Stub>
}`

Of course you can write just like that ` json = {"stub": stub.as_structure()}` in `server.add_stub` 
but I tried  to keep project style.

Reply:
@brunns Do you think it is better to remove this class?

Reply:
We could just have `index` as an optional argument in the `add_stub()` method, perhaps?

Reply:
Like that?
`    def add_stubs(self, definition: Union[Stub, Iterable[Stub]], index: int = None):
        if isinstance(definition, abc.Iterable):
            for stub in definition:
                self.add_stubs(stub)
        else:
            json = {"stub": definition.as_structure()}
            if index is not None:
                json["index"] = index`

Reply:
Like that?
```    def add_stubs(self, definition: Union[Stub, Iterable[Stub]], index: int = None):
        if isinstance(definition, abc.Iterable):
            for stub in definition:
                self.add_stubs(stub)
        else:
            json = {"stub": definition.as_structure()}
            if index is not None:
                json["index"] = index```

Reply:
Or 
`json = AddStub(stub=definition, index=index).as_structure()`

Reply:
Made second variant in  #36